.vscode/settings.json

```diff
@@ -3,6 +3,7 @@
     "prettier.printWidth": 120,
     "prettier.singleQuote": true,
     "prettier.tabWidth": 4,
+    "scss.lint.unknownProperties": "ignore",
```

Comment:
Is this for `vector-effect`? Should it be applied directly to the path elements as an attribute, instead?

Reply:
Yeah, sure